modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/java/com/liferay/dynamic/data/mapping/web/internal/exportimport/data/handler/DDMStructureStagedModelDataHandler.java

```diff
@@ -543,6 +549,59 @@ private void _exportDDMFormLayout(
 			ddmFormLayoutPath, structureLayout.getDefinition());
 	}
 
+	private void _exportReferencedDDMStructure(
+		PortletDataContext portletDataContext, DDMStructure structure,
+		DDMFormField ddmFormField) {
+
+		long ddmStructureId = GetterUtil.getLong(
+			ddmFormField.getProperty("ddmStructureId"));
+
+		DDMStructure referencedStructure =
+			_ddmStructureLocalService.fetchDDMStructure(ddmStructureId);
+
```

Comment:
@mbowerman could you rename from `referencedStructure` to `referencedDDMStructure`? Thanks!

Reply:
@carolmariaabb Done